AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +651,65 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    default_dash_layout = '''
+        {"nav" :
+              {"App Management":{"True":
+                {"Upload App":"/apps/new",
+                "Delete App":"/apps/delete"}},
+              "AppScale Management":{"True":
+                {"Cloud Status":"/status/cloud",
+                "Manage Users":"/authorize"}},
+              "Debugging/Monitoring":{"True":
+                {"Monit":"temp1",
+                "TaskQueue":"temp2",
+                "Logging":"/logs"}}}
```

Comment:
so that "True" is eventually is_user_cloud_admin or can_upload_apps or different combinations of those two. If I manage it in the template I have to test which heading I have to see which variables I need to use.